tox.ini

```diff
@@ -20,10 +20,26 @@ commands = nosetests --ignore-files=test_appengine\.py {posargs}
 [testenv:cover]
 basepython = python2.7
 commands =
-    nosetests --with-xunit --with-xcoverage --cover-package=oauth2client --nocapture --cover-erase --cover-tests --cover-branches --ignore-files=test_appengine\.py --cover-min-percentage=60
+    nosetests \
```

Comment:
Yep. the ``--x`` flags are gone because nosexcover is gone. ``--nocapture`` really isn't needed here, as there's no point in having everything print to stdout for the coverage run. ``--cover-min-percentage`` is sadly useless with multiple runs. You can maybe run coverage itself afterwards if you still wanted the coverage test to fail if coverage goes under a certain percentage.